lib/draft_generators.rb

```diff
@@ -3,6 +3,7 @@
 require "devise_customization_service"
 require "rails_tag_service"
 require "devise"
+require 'indefinite_article'
```

Comment:
Looks like we're using double-quotes in the rest of the file, maybe use them here also for consistency?

Or, I wonder what running `rubocop -a` would do in this project, to autoformat all files.

Reply:
Yep, I can change this.  I didn't try to run a `rubocop -a`, after I make the commit I will revert it back to the double quotes on my computer and see if rubocop will catch this.